src/components/ui/table.tsx

```diff
@@ -5,7 +5,7 @@ const Table = forwardRef<
     HTMLTableElement,
     React.HTMLAttributes<HTMLTableElement>
 >(({ className, ...props }, ref) => (
-    <div className="relative w-full overflow-auto rounded border border-secondary-light dark:border-secondary-dark">
+    <div className="tableScrollbar relative w-full overflow-auto rounded border border-secondary-light dark:border-secondary-dark">
```

Comment:
`table-scrollbar`

Reply:
Infact, we should style the generic scrollbar instead of creating a class just for the tables - using the theme config

Reply:
Resolved in: [c7f532c](https://github.com/covalenthq/goldrush-kit/pull/186/commits/c7f532c7a1c7bc29a1efc773c18b824796887f73)